src/js/utils/rxjsUtils.js

```diff
@@ -2,18 +2,29 @@ import { timer } from "rxjs/observable/timer";
 
 import "rxjs/add/operator/delayWhen";
 import "rxjs/add/operator/scan";
+// import { Observable } from "rxjs/Observable";
+// import { Scheduler } from "rxjs/Scheduler";
 
 /* eslint-disable import/prefer-default-export */
+/**
+ * linearBackoff callback to use with e.g. delayWhen
+ *
+ * @param  {number} delay
+ * @param  {number} maxRetries
+ * @param  {number} [maxDelay=Number.MAX_SAFE_INTEGER]
+ * @param  {Scheduler} [scheduler=null]
+ * @return {Observable} notifier obervable
+ */
 export function linearBackoff(
-  maxRetries = 5,
-  delay = 1000,
+  delay,
+  maxRetries,
```

Comment:
I made this argument required, which is the same behavior as in `retry()`

Reply:
I think retry is actually defaulting to -1.

```
export function retry<T>(count: number = -1): MonoTypeOperatorFunction<T> {
  return (source: Observable<T>) => source.lift(new RetryOperator(count, source));
}
```
https://github.com/ReactiveX/rxjs/blob/53b51f35689278e05e400506dbc7388a6df5168e/src/internal/operators/retry.ts

Reply:
narf… was just looking at the documentation, not in the code…